content/docs/user-guide/gems/reference/environment/surface-data.md

```diff
@@ -6,4 +6,19 @@ weight: 400
 toc: true
 ---
 
-The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth.
+The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth or terrain.

```

Comment:
minor suggestion to add additional information.

```suggestion
The Surface Data Gem enables surfaces and meshes to emit signals or tags that communicate various attributes. These signals and tags have various uses such as creating inclusion and exclusion areas, aligning and positioning objects such as vegetation, and applying physics materials.
```

Reply:
Sounds good!